routes/boundary.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+from flask import (
+    abort,
+    Blueprint,
+    Response,
+    render_template,
+    request,
+    current_app as app,
+)
+
+# local imports
+from . import routes
+from luts import huc8_gdf, akpa_gdf
+
+boundary_api = Blueprint("boundary_api", __name__)
+
+
+@routes.route("/boundary/")
+@routes.route("/boundary/abstract/")
+def boundary_about():
+    return render_template("boundary/abstract.html")
+
+
+@routes.route("/boundary/protectedarea/")
+@routes.route("/boundary/protectedarea/abstract/")
+def protectedarea_about():
+    return render_template("boundary/protectedarea.html")
+
+
+@routes.route("/boundary/huc/")
+@routes.route("/boundary/huc/abstract/")
+@routes.route("/boundary/watershed/")
+@routes.route("/boundary/watershed/abstract/")
+def huc_about():
+    return render_template("boundary/huc/abstract.html")
+
+
+@routes.route("/boundary/watershed/huc8/")
+@routes.route("/boundary/huc/huc8/")
+def huc8_about():
+    return render_template("boundary/huc/huc8.html")
+
+
+@routes.route("/boundary/huc/huc8/<huc8_id>")
+@routes.route("/boundary/watershed/huc8/<huc8_id>")
+def run_fetch_huc_poly(huc8_id):
+    """Run the async requesting for a HUC polygon and return the GeoJSON.
+
+    Args:
+        huc8_id (int): HUC-8 ID
+
+    Returns:
+        GeoJSON of the HUC-8 polygon
+
+    Notes:
+        example: http://localhost:5000/boundary/huc/huc8/19070506
+    """
+    poly = huc8_gdf.to_crs(4326).loc[[huc8_id]]
```

Comment:
sweet! I'm gonna push a little change to swap the order of operations here from reproject/query to query reproject, it's doing a bit more than it needs as it reprojects the entire geodataframe for each query.